menus/tree-view.cson

```diff
@@ -97,4 +97,9 @@
   'atom-pane[data-active-item-path] .tab.active': [
     {'label': 'Rename', 'command': 'tree-view:rename'}
     {'label': 'Reveal in Tree View', 'command': 'tree-view:reveal-active-file'}
+    {'label': 'Show In File Manager', 'command': 'tree-view:show-current-file-in-file-manager'}
+  ]
+
+  'atom-text-editor': [
```

Comment:
Should probably be `atom-text-editor:not([mini])` so that this doesn't appear in mini-editors.